test/new_routes_test.go

```diff
@@ -1086,7 +1092,9 @@ func TestNewRouteStreamImport(t *testing.T) {
 	sendB("UNSUB 1\r\nPING\r\n")
 	expectB(pongRe)
 
-	checkExpectedSubs(0, srvA)
+	if err := checkExpectedSubs(0, srvA); err != nil {
```

Comment:
Did this not t.Fatalf on its on?

Reply:
Not the one in the `/test` directory. The one in `/server` accepts `t` and so calls t.Fatalf(), but not this one. I could fix the 37 occurrences in `/test` package. You want me to do that?

Reply:
nah.. all good.